python/ray/dashboard/client/src/api.ts

```diff
@@ -185,12 +190,11 @@ export type FullActorInfo = {
   webuiDisplay?: Record<string, string>;
 };
 
-export type PartialActorInfo = {
+export type ActorTaskInfo = {
   actorId?: string;
   actorTitle?: string;
   requiredResources?: { [key: string]: number };
-  state: ActorState.Invalid;
-  invalidStateType?: InvalidStateType;
+  state: ActorState.Infeasible | ActorState.PendingResources;
```

Comment:
It no longer has to receive null. I changed the API behavior because the dashboard calculates this state itself. It will always be set for an actor creation task, which is what this data type represents.